Crawler/Configs/DataFilesConfig.cs

```diff
@@ -6,5 +6,6 @@ public class DataFilesConfig
 		public string EmotionsFile { get; set; }
 		public string NumbersFile { get; set; }
 		public string QuestionsFile { get; set; }
+		public string ToBeFormsFile { get; set; }
```

Comment:
to be or not to be that is the question
(ToBeTenses)

Reply:
I'm sure you enjoyed writing it, but it looks like "ToBeForms" is the right form ;).